reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md

```diff
@@ -144,20 +151,22 @@ permitted.
 
 ### HOW TO ENABLE REMOTING ON PUBLIC NETWORKS
 
+```
 ERROR:  Unable to check the status of the firewall
+```
 
-The Enable-PSRemoting cmdlet returns this error when the local network
+The `Enable-PSRemoting` cmdlet returns this error when the local network
 is public and the SkipNetworkProfileCheck parameter is not used in the
 command.
 
-On server versions of Windows, Enable-PSRemoting succeeds on all network
+On server versions of Windows, `Enable-PSRemoting` succeeds on all network
 location types. It creates firewall rules that allow remote access to
 private and domain ("Home" and "Work") networks. For public networks, it
 creates firewall rules that allows remote access from the same local subnet.
 
-On client versions of Windows, Enable-PSRemoting succeeds on private and
+On client versions of Windows, `Enable-PSRemoting` succeeds on private and
 domain networks. By default, it fails on public networks, but if
-you use the SkipNetworkProfileCheck parameter, Enable-PSRemoting succeeds
+you use the SkipNetworkProfileCheck parameter, `Enable-PSRemoting` succeeds
```

Comment:
SkipNetworkProfileCheck should be bolded.

Reply:
fixed